roles/softserve/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Install required packages
+  package:
+    state: present
+    name: "{{ item }}"
+  with_items:
+  - python-virtualenv
+  - gcc
+  - git
+  - python-devel
+  - postgresql-devel
+
+- name: Add tmpfiles snippet
+  template:
+    dest: /etc/tmpfiles.d/softserve.conf
+    src: softserve.tmpfiles.conf
+  notify: Create systemd tmpfile
+
+- name: Add gunicorn systemd service
+  template:
+    dest: /etc/systemd/system/softserve.service
+    src: softserve.service
+  notify: Reload systemd
+
+- name: Start the softserve service
+  service:
+    name: softserve
+    enabled: True
+    state: started
+
+- name: Create the project directory
+  file:
+    path: '/home/softserve'
```

Comment:
I would prefer it outside of /home (like /srv), and in a variable. Outside of /home would avoid various selinux side effect.